.github/workflows/deploy.yml

```
@@ -0,0 +1,42 @@
+    - name: Install Wrangler
+      run: npm install -g wrangler
+    
+    - name: Deploy to Cloudflare Pages
+      uses: cloudflare/pages-action@v1
+      with:
+        apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+        accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+        projectName: blue-badge-app
+        directory: .
+        gitHubToken: ${{ secrets.GITHUB_TOKEN }}
+        environment: production
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)